src/app/components/Addcard/AddCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styles from './AddCard.module.css';
+import Typography from '../Typography/Typography';
+import PlusIcon from '../assets/PlusIcon';
+
+export type AddCardProps = {
+  imageSrc: string;
+  title: string;
+  description: string;
+  onClick?: () => void;
+};
+
+const AddCard = ({
+  imageSrc,
+  title,
+  description,
+  onClick,
+}: AddCardProps): JSX.Element => {
+  return (
+    <article className={styles.card}>
+      <img src={imageSrc} alt="placeholder image" className={styles.imgage} />
+      <div className={styles.content}>
+        <Typography size="m">{title}</Typography>
+        <Typography size="p">{description}</Typography>
+      </div>
+      <button onClick={onClick} className={styles.addButton}>
+        <PlusIcon />
+      </button>
+    </article>
+  );
+};
```

Comment:
Not sure, because I don't know the rest of the code but seeing the component in `storybook` this feels like that the `article` should be wrapped in a `li` element. And then render a `List` of `Cards`. But just a feeling. ;-)

Reply:
Yes, you are right. I fixed it asap. I hope the semantic is okay now!